nixos/modules/config/ldap.nix

```diff
@@ -257,6 +257,8 @@ in
     systemd.services = mkIf cfg.daemon.enable {
       nslcd = {
         wantedBy = [ "multi-user.target" ];
+        wants = [ "network-online.target" ];
```

Comment:
Can we set this only if ldap logins are used. I don't want to slow down the startup for all NixOS users.

Reply:
Confused it with nscd.